tests/helpers/linux/test_pid.py

```diff
@@ -30,5 +31,23 @@ def test_find_task(self):
         self.assertEqual(task.comm.string_(), comm)
 
     def test_for_each_task(self):
-        pid = os.getpid()
-        self.assertTrue(any(task.pid == pid for task in for_each_task(self.prog)))
+        NUM_PROCS = 12
+        barrier = Barrier(NUM_PROCS + 1)
```

Comment:
It looks like we might need to mount `/dev/shm` in the vmtest setup for this to work?

Reply:
It works on my machine when I run vmtest for one of the kernels that is failing with the `/dev/shm` error on CI (e.g. `4.4.293`), so I think this has something to do with the CI setup.

Reply:
Fixed by https://github.com/osandov/drgn/commit/0f68cd44e296b0b2a56d20c58da55a2998a80bfe